src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -1323,4 +1344,39 @@ private bool IsValidPermissionAction(EntityActionOperation action, Entity entity
             return action is EntityActionOperation.All || EntityAction.ValidPermissionOperations.Contains(action);
         }
     }
+
+    /// <summary>
+    /// Returns whether the log-level keyword is valid or not.
+    /// It does this by checking each section of the name of the class
+    /// E.g. Azure.DataApiBuilder is valid. While Azure.DataA is invalid.
+    /// </summary>
+    /// <param name="loggerFilter">String keyword that comes from log-level in config file</param>
+    private static bool IsLoggerFilterValid(string loggerFilter)
+    {
+        string[] loggerSub = loggerFilter.Split('.');
+        for (int i = 0; i < LoggerFilters.validFilters.Count; i++)
+        {
+            bool isValid = true;
+            string[] validFiltersSub = LoggerFilters.validFilters[i].Split('.');
```

Comment:
if the `LoggerFilters.validFilters` are used in `Core`  why can't we define it inside Core, why does it need to live in the Config project?